angular/src/app/admin/dam-repository/resources/resource-form/resource-form.component.ts

```diff
@@ -157,6 +167,10 @@ export class ResourceFormComponent implements OnInit, AfterViewInit, Form {
     return [...this.getViewChildrenForms(), this.form];
   }
 
+  viewFormChange() {
```

Comment:
this should be renamed and reused on line `83` e.g. `formChange` (since view form is part of this form we can just call it generic formChange, since the effect will be the same)